src/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  display: flex;
+  background-color: $bg-color;
+
+  &__item {
+    margin: 0 10px;
+    text-decoration: none;
+    list-style: none;
+  }
+
+  &__link {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    position: relative;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    color: #060b35;
```

Comment:
Consider using a variable for the color `#060b35` to maintain consistency with other color definitions and improve maintainability.